qemu-guest-agent.service

```diff
@@ -9,7 +9,7 @@ EnvironmentFile=/etc/sysconfig/qemu-ga
 ExecStart=/usr/bin/qemu-ga \
   --method=vsock-listen \
   --path=3:1234 \
-  --blacklist=${BLACKLIST_RPC} \
+  $FILTER_RPC_ARGS \
```

Comment:
The RHEL systemd unit uses `${FILTER_RPC_ARGS}`, otherwise fine with me.

Reply:
https://issues.redhat.com/browse/RHEL-116179